### DISCUSSION TITLE
Not able to control brightness for HDMI over type C

### DISCUSSION THREAD

**Discussion by sharique** [emoacht/Monitorian#293] [Compatibility]
I'm using a multi-port type c adapter to connect to monitor using HDMI port of adapter, but it is not detected by the application.

**Comment by emoacht** [comment]
What is your adapter model?
I guess that adaptor is not compatible with DDC/CI but your comment  has no meaningful information.

**Comment by sharique** [comment]
hi @emoacht I'm using adapter from Cadyce, you can check it here
https://www.amazon.in/Gigabit-Ethernet-Adapter-Charging-CA-CU3HVG/dp/B07QWTJPQY/

**Comment by emoacht** [comment]
Thanks for the information.
I am transfering this to discussion for future reference.
